rollup.config.js

```diff
@@ -29,13 +32,23 @@ const config = mode => ({
         if (source === 'ReactDOM') {
           return {id: 'react-dom', external: true};
         }
+        if (source === 'ReactNative') {
```

Comment:
I get that this is following prior art, but this seems like an artifact that we could remove unless there is a compelling reason to keep the PascalCase aliases.

Reply:
Resolving because we need to preserve PascalCase imports